docs/contributing-to-airbyte/developing-locally.md

```diff
@@ -80,6 +80,32 @@ The build will take a few minutes. Once it completes, Airbyte compiled at curren
 
 In `dev` mode, all data will be persisted in `/tmp/dev_root`.
 
+## Run the `dev` image of a connector
+
+These instructions explain how to run a version of an Airbyte connector that you are developing on (e.g. has not been released yet).
+
+- First, start the web app:
+
+```bash
+SUB_BUILD=PLATFORM ./gradlew build
+VERSION=dev docker-compose up
+```
+
+- Then, build the connector image:
+```
+./gradlew :airbyte-integrations:connectors:<connector_name>:airbyteDocker
+```
+When building via Gradle, the docker image name and tag, respectively, are the values of the `io.airbyte.name` and `io.airbyte.version` `LABEL`s in
+the Dockerfile.
```

Comment:
I'd suggest sharing the following command, this will be faster than running the Gradle command and the developer can use the tag they like
```
docker build ./airbyte-integrations/connectors/<connector-name> -t airbyte/<connector-name>:dev
```